WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+- [Other](#other)
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy new work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
```

Comment:
This should address relationship between IPFS and libp2p, or the document should give an overarching statement about the relationship between IPFS work and libp2p work, and how that impacts the scope of each working group.